clang/include/clang/Format/Format.h

```diff
@@ -5230,6 +5239,7 @@ struct FormatStyle {
            TableGenBreakingDAGArgOperators ==
                R.TableGenBreakingDAGArgOperators &&
            TableGenBreakInsideDAGArg == R.TableGenBreakInsideDAGArg &&
+           TabWidth == R.TabWidth && TemplateNames == R.TemplateNames &&
```

Comment:
You duplicated TabWidth. (Sorry for the late review. ;))

Reply:
Fixed in 04f14e4d652a344fedea6de8df532e88a652bcc4.